app/main.py

```diff
@@ -1,3 +1,15 @@
 def cache(func):
-    # Write your code here
-    pass
+    count = {}
+
+    def wrapper(*args):
+        nonlocal count
+
+        if args in count.keys():
```

Comment:
```suggestion
        if args in count:
```
This statement could be simplified